src/Admin.php

```diff
@@ -32,6 +32,7 @@ public static function init() {
     // Exposes SVG images in media library.
     add_filter('wp_prepare_attachment_for_js', __CLASS__ . '::wp_prepare_attachment_for_js');
     add_action('admin_head', __CLASS__ . '::admin_head');
+    add_filter('image_send_to_editor', __NAMESPACE__ . '\Plugin::removeSvgWidthHeightAttributes', 10, 2);
```

Comment:
1. The added line should be separated with a blank line and an own inline comment that explains what it does, because it does not belong to exposing SVG images in the media library.

1. The registration of this filter callback can be moved to the other registered filters in `Plugin::init()`, so that everything is in one place.  (When we talked about it, I assumed there to be more than just the registration.)